processes/burntmapping_chunks.json

```
@@ -0,0 +1,196 @@
+{
+  "namespace": "user",
+  "description": "Classical Normalized Burnt Ratio(NBR) difference is performed using VITO backend on a chunk polygon.",
+  "id": "NBR_chunk_polygons",
+  "summary": "Burnt area mapping using chunk_polygon on UDF",
+  "returns": "Return an RGB image as Geotiff",
+  "links": [
+    {
+      "href": "https://openeo.vito.be/openeo/1.1/processes/u:ecce9fea04b8c9c76ac76b45b6ba00c20f211bda4856c14aa4475b8e8ed433cd%40egi.eu/NBR_chunk_polygons",
+      "rel": "canonical",
+      "title": "Public URL for user-defined process 'NBR_chunk_polygons'"
+    }
+  ],
+  "parameters": [
+    {
+      "description": "Left-closed temporal interval, i.e. an array with exactly two elements:The first element is the start of the temporal interval. The specified instance in time is included in the interval.The second element is the end of the temporal interval. The specified instance in time is excluded from the interval.The specified temporal strings follow RFC 3339. Also supports open intervals by setting one of the boundaries to null, but never both.",
```

```suggestion
      "description": "Left-closed temporal interval, i.e. an array with exactly two elements: The first element is the start of the temporal interval. The specified instance in time is included in the interval. The second element is the end of the temporal interval. The specified instance in time is excluded from the interval. The specified temporal strings follow RFC 3339. Also supports open intervals by setting one of the boundaries to null, but never both.",
```

Generally, please use spaces between dots and the next word.

---

Oops. Sorry about those. The typos are corrected now.